src/Microsoft.DotNet.Darc/src/DarcLib/GitHubClient.cs

```diff
@@ -351,23 +351,42 @@ public async Task UpdatePullRequestAsync(string pullRequestUri, PullRequest pull
                 });
         }
 
+
+        public async Task<IList<Commit>> GetPullRequestCommitsAsync(string pullRequestUrl)
+        {
+            (string owner, string repo, int id) = ParsePullRequestUri(pullRequestUrl);
+            Octokit.PullRequest pr = await Client.PullRequest.Get(owner, repo, id);
```

Comment:
is the id that we parse from the URL different from the pr.Number that you get from getting the PR? Wondering if this call is unnecessary and you can just pass the id to when you're getting the commits.